modules/toyvolume/rendering/renderabletoyvolume.cpp

```diff
@@ -141,7 +141,8 @@ RenderableToyVolume::RenderableToyVolume(const ghoul::Dictionary& dictionary)
 RenderableToyVolume::~RenderableToyVolume() {}
 
 void RenderableToyVolume::initializeGL() {
-    _raycaster = std::make_unique<ToyVolumeRaycaster>(_color);
+    glm::vec4 color{ glm::vec3(_color), _opacity };
```

Comment:
Personally, I'm not a big fan of using the new syntax initialization as it is more susceptible to errors when the struct signature changes, so I'd prefer
`glm::vec4 color(glm::vec3(_color), _opacity);` in this case